tests/vcd_role.py

```diff
@@ -26,5 +41,13 @@ def test_get_rights(self):
         rights = role.list_rights()
         assert len(rights) > 0
 
+    def test_create_role(self):
+        logged_in_org = self.client.get_org()
+        org = Org(self.client, resource=logged_in_org)
+        role_name = self.config['vcd']['role_name']
+        org.create_role(role_name, 'test role', {'vApp: Copy'})
```

Comment:
If the method is expecting a tuple of strings, this should be `, ('vApp: Copy'))`, not a dict

Reply:
my mistake, fixed.